learning_orchestra_client/utils/file_caster.py

```diff
@@ -0,0 +1,52 @@
+from os import PathLike
+
+
+def python_file_caster(source: str, filename: str = '<string>',
```

Comment:
Maybe there is a name more descritive about what this function does, i think that 'python_file_caster' doesn't sound compreensive to user.

Reply:
maybe 'code_reader' can be more suggestive, you also can request suggestions on project team,